### DISCUSSION TITLE
TPI coeff different for heat and cooling & 1 switch for heat and 1 switch for cooling

### DISCUSSION THREAD

**Discussion by razorback91** [jmcollin78/versatile_thermostat#466] [Ideas]
Hello,

I'm migrating from jeedom to homeassistant and the thermostat in jeedom allow to select different TPI coeff for heating and cooling. More over it is possible to have 1 switch for heating and 1 switch for cooling.

I will work around with 2 thermostats but if we could have 2 in 1 it will make the configuration more simple.

Thank you

Xavier

**Comment by jmcollin78** [comment]
VTherm is already doing the 2 in 1 you are claming. I use this myself on my reversible Hvac device.

**Reply by razorback91** [reply]
yes but I want to use in a different way. Actaully I love the PID calculation and it is much more precise than the control my AC can do. What I do today with my jeedom in thermostat I have :
1/ Heat : Coeff, action to be done to heat, action to be done to stop
2/ Cooling : other Coeff than heat, action to be done to cool and action to stop.

I defined in 2 presets on my AC like a button.

**Reply by jmcollin78** [reply]
I understand your point and this will lead to over complexity in configuration for a finally very small gain.
So I do not intend to implement this.

**Comment by razorback91** [comment]
the gain is to have a temperature in cooling mode as precise as when it is in heating mode.
I don't want to rely on the cooling system to control my temp. I prefer by far the PID algo.
No big deal, I will implement 2 thermostats. 1 for cooling and 1 for heating.

Thank you

**Reply by razorback91** [reply]
Hello, do you believe TPI coeff could be changed in a service ?

**Reply by jmcollin78** [reply]
Hello, this could be an option yes.
